tasks/MessageDecoder.hpp

```diff
@@ -62,6 +62,16 @@ namespace comms_webrtc
             return jdata["data"]["candidate"].asString();
         }
 
+        bool validateMidFieldPresent()
```

Comment:
It's not `validating` the way `validateFieldPresent` work. This will be confusing. Rename to `isMidFieldPresent`